src/themes/dojo/variables.css

```diff
@@ -55,4 +49,24 @@
 	--calendar-disabled-bg: var(--dojo-light-gray);
 	--calendar-hover-bg: var(--dojo-light-blue);
 	--calendar-selected: var(--dojo-blue);
+
+	/* New theme variables */
+	--dojo-black: #000000;
+	--dojo-dark-gray: #5C6C7C;
+	--dojo-gray: #D6DDE2;
+	--dojo-light-gray: #F4F6F7;
+	--dojo-white: #FFFFFF;
+
+	--pane-bg: var(--dojo-white);
+	--border-color: var(--dojo-light-gray);
+	--base-color: var(--dojo-dark-gray);
+
+	--base-font-size: 16px;
+	--base-line-height: 24px;
+	--border-thin: 1px;
+	--drop-shadow: 0 9px 16px 0 rgba(0, 0, 0, 0.5);
+	--grid-size: 8px;
+	--header-font-size: 20px;
+	--header-line-height: 1.2;
+	--underlay-bg: rgba(0, 0, 0, 0.59);
```

Comment:
0.6, maybe? :)
Could also use `color(var(--dojo-black) alpha(60%))` for this and `--drop-shadow`